bin/express

```diff
@@ -398,18 +448,20 @@ function main() {
   var appName = createAppName(path.resolve(destinationPath)) || 'hello-world'
 
   // View engine
-  if (program.view === undefined) {
+  if (program.view) {
```

Comment:
Is the desired behavior to allow conflicting views to be specified and give preference to the more recent format? Would it be better to blow up, as it seems like it's obviously an error?